main/guides/zoe/contract-basics.md

```diff
@@ -58,7 +58,7 @@ Ordinary heap state persists between contract invocations.
 
 We'll discuss more explicit state management for
 large numbers of objects (_virtual objects_) and
-objects that last across upgrades ([durable objects](./contract-upgrade.md#durability)) later.
+objects that last across upgrades ([durable objects](./contract-upgrade#durability)) later.
```

Comment:
odd... links don't go to `xyz.md` anymore?

Reply:
They do, but the the extensions are now optional so I chose to remove them for simplicity.

Reply:
Hm. simplicity vs. consistency...

CONTRIBUTING still says
> Next, your Markdown links should be to the `.md` Markdown files ...

So unless we aim to change the shop norm, I prefer consistency. (not critical)

Reply:
The wording of that sentence is a bit confusing to me. Updated here [37b292d](https://github.com/Agoric/documentation/pull/971/commits/37b292d594cbb6ab9996bce41ce45cb6375201dd)